torchsynth/module.py

```diff
@@ -620,13 +620,18 @@ class Noise(SynthModule):
     these in a synth then choose different seeds for each instance.
     """
 
+    # Do we really want deterministic noise within each batch?
+    # https://github.com/turian/torchsynth/issues/250
     noise_batch_size: int = 64
     # Unfortunately, Final is not supported until Python 3.8
     # noise_batch_size: Final[int] = 64
 
     def __init__(self, synthconfig: SynthConfig, seed: int, **kwargs):
         super().__init__(synthconfig, **kwargs)
 
+        # https://github.com/turian/torchsynth/issues/255
+        assert self.batch_size == 64 or not self.synthconfig.reproducible
+
```

Comment:
Let's dump all the offset stuff and just return the noise now that we are enforcing a batch size of 64 when reproducible

Reply:
@jorshi make a separate PR. But keep in mind we're switching to batch_size 128 maybe